crates/parcel_plugin_rpc/src/nodejs/host.rs

```diff
@@ -17,11 +17,11 @@ use super::RpcWorkerNodejs;
 // RpcHostNodejs has a connection to the main Nodejs thread
 pub struct RpcHostNodejs {
```

Comment:
As inverted this ;we'll always suffix trait implementations with the trait name.

This should be `NodejsRpcHost`

Reply:
Noted